doc/user_manual/postprocessor.tex

```diff
@@ -1226,6 +1226,7 @@ \subsubsection{Interfaced}
         \item \textbf{min}, compute the minimum of each variable along each single history
          \item \textbf{max}, compute the maximum of each variable along each single history
          \item \textbf{average}, compute the average of each variable along each single history
+         \item \textbf{all}, join together all of the each variable in the history, and make the pivotParameter as an input parameter. (So if the original HistorySet had input parameters x,y, a pivotParameter of t, and output: a,b then the new input parameters would be x, y, t and the output would be a, b)
```

Comment:
can you break the line?

Is the fact that "t" becomes an input parameter relevant or needed? shouldn't be a user decision via the settings in the Output (PS) data object?

For example, 
if I have "t" in <Input> it goes there, or in <Output> otherwise.

Reply:
@joshua-cogliati-inl

Reply:
In the current implementation, the fact that 't' becomes an input parameter is very relevant, since otherwise, how does the user know what to put in the Data Object specification.  Also, if "t" is not part of the input space, it is very easy to get duplicate inputs.

Reply:
Right now the implementation does not look at the output data object, it just figures out what is needed from the input data.  Do you want to re-look at the implementation and see if it needs to be modified to check the output data object?

Reply:
I checked, and which end up as input and which as output depends on the DataObject specification.  I expanded the explanation and hopefully it is now clearer.